build.gradle

```diff
@@ -125,6 +125,7 @@ jar {
 
     manifest {
         attributes 'ContainedDeps': coremodJar.archivePath.name
+        attributes 'Main-Class': 'net.montoyo.mcef.setup.SetupUI'
```

Comment:
this is new: added a default Main-Class so when you double-click the .jar file it runs the setup menu again (was broken before).